crates/polars-utils/src/functions.rs

```diff
@@ -1,4 +1,18 @@
 use std::hash::{BuildHasher, Hash};
+use std::ops::Range;
+
+// The ith portion of a range split in k (as equal as possible) parts.
+#[inline(always)]
+pub fn range_portion(i: usize, k: usize, r: Range<usize>) -> Range<usize> {
```

Comment:
Not yet used I believe.

Reply:
Correct, I can remove it if you prefer just wanted to have it ready for future PRs.